lib/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior.rb

```diff
@@ -16,16 +15,6 @@ module PcdmValkyrieBehavior
         #           Since member_ids is all that keeps these in wings, how can we do that here?
       end
 
-      # Add member objects by adding this collection to the objects' member_of_collection association.
```

Comment:
Originally defined in Hyrax::CollectionBehavior so moved to a separate file that mirrors that file.